Gateway/Response/PaymentPosCloudHandler.php

```diff
@@ -81,5 +91,19 @@ public function handle(array $handlingSubject, array $response)
         // do not close transaction so you can do a cancel() and void
         $payment->setIsTransactionClosed(false);
         $payment->setShouldCloseParentTransaction(false);
+
+        if (in_array($resultCode, ['Authorised', 'Success'])) {
```

Comment:
Terminal API doesn't return `Authorised` result code.

```suggestion
        if ($resultCode === PaymentResponseHandler::POS_SUCCESS) {
```